benchmarking/runs/concentrate_gpa.py

```diff
@@ -92,7 +94,9 @@ def start(self, num_trials: int = 10, generate_graphs: bool = True):
                         PriorityAlgorithmConfig(),
                         PriorityAlgorithmConfig(
                             name="local_max",
-                            MUTATIONS=[(mutate_local_max, 1), (mutate_random_swap, 2)],
+                            MUTATIONS=MutationSet(
+                                [(LocalMaxMutation(), 1), (RandomSwapMutation(), 2)]
```

Comment:
Wait wouldn't it be more like `[LocalMaxMutation(1), RandomSwapMutation(2)]` now?